docs/rapidast-manual-scan.md

```
@@ -0,0 +1,228 @@
+
+- The `requestor` field is only the email destination for the key file. It does **not** grant browser access to the GCS bucket.
+- To get browser access (view results in the GCS console), you **must** set the `group` field to a valid Google Group email. Anyone in that group will have access to browse the folder.
+- If you forget the `group` field, you can still upload results via RapiDAST (using the key file), but you won't be able to view them in the browser. Submit a follow-up MR to add the group.
+
+Reference: [Exporting RapiDAST scan results to ProdSec central storage](https://redhat.atlassian.net/wiki/spaces/PRODSEC/pages/289244711)
+
+## GCS Folder Structure
+
+Results must follow this structure, with one subfolder per release version:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)